policy-test/tests/outbound_api_gateway.rs

```diff
@@ -10,6 +10,11 @@ use linkerd_policy_test::{
 use maplit::{btreemap, convert_args};
 use tokio::time;
 
+// These tests are copies of the tests in outbound_api_gateway.rs but using the
+// policy.linkerd.io HttpRoute kubernetes types instead of the Gateway API ones.
```

Comment:
```suggestion
// These tests are copies of the tests in outbound_api_linkerd.rs but using the
// gateway.networking.k8s.io HttpRoute kubernetes types instead of the Linkerd API ones.
```